test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -886,5 +887,50 @@ void TestNormalizeLogMeanVarianceFixZeroVec()
             Assert.Equal(0f, transformedDataArray[2].Features[1]);
             Assert.Equal(0f, transformedDataArray[2].Features[4]);
         }
+
+        public class TensorData
+        {
+            private const int dim1 = 2;
+            private const int dim2 = 3;
+            private const int dim3 = 4;
+            private const int size = dim1 * dim2 * dim3;
+
+            [VectorType(dim1, dim2, dim3)]
+            public float[] input { get; set; }
+
+            public static TensorData[] GetTensorData()
+            {
+                var tensor1 = Enumerable.Range(0, size).Select(
+                x => (float)x).ToArray();
+
+                var tensor2 = Enumerable.Range(0, size).Select(
+                x => (float)(x + 10000)).ToArray();
+
+                return new TensorData[]
+                {
+                    new TensorData() { input = tensor1},
+                    new TensorData() { input = tensor2}
+                };
+            }
+        }
+
+        [Fact]
```

Comment:
When fixing a bug, it is generally useful to add the repro scenario as a test case. That way it ensures the full scenario is actually fixed, and won't regress again in the future.

Reply:
So, do you mean that I should remove this test case and replace it with the problem described in the original issue (including a small dataset of images, and using the resnet18)?

Reply:
I’m not necessarily saying remove this test. It can still provide value by itself testing at a lower level.

But I’m saying it is good to have the complete end-to-end scenario tested as well. 

We should already have an image set to use from other image tests.

Reply:
I've added this test in my latest iteration. Thanks.